ompi/mca/pml/ob1/pml_ob1_recvreq.h

```diff
@@ -268,7 +271,7 @@ static inline void recv_req_matched(mca_pml_ob1_recv_request_t *req,
  *
  */
 
-#define MCA_PML_OB1_RECV_REQUEST_UNPACK( request,                                 \
```

Comment:
Why this change ?

Reply:
It looks there might be a conflict with the .request field in the mca_pml_ob1_transfer_event_t struct.